Unix/installbuilder/service_scripts/omid.upstart

```diff
@@ -0,0 +1,54 @@
+##
+## Copyright (c) Microsoft Corporation.  All rights reserved.
+##
+## Contains settings for the OMI WS-Management Deamon.
+##
+##
+#
+#### BEGIN INIT INFO
+## Provides:          omid
+## Required-Start:    $network
```

Comment:
And file system, if there's a service for that. We needed that on AIX, I believe.